src/python/bot/fuzzers/libfuzzer.py

```diff
@@ -376,9 +375,17 @@ class FuchsiaQemuLibFuzzerRunner(new_process.ProcessRunner, LibFuzzerCommon):
 
   FUZZER_TEST_DATA_REL_PATH = os.path.join('test_data', 'fuzzing')
 
-  def _setup_fuzzer_and_device(self):
-    """ Build a Fuzzer object based on the QEMU values.
-    Call this only after setup_qemu_values()"""
+  def _setup_device_and_fuzzer(self, qemu_is_running=True):
```

Comment:
instead of adding this condition here, just create another separate method which only starts qemu and use it in the places where you'd normally pass qemu_is_running=False